website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -80,6 +72,19 @@ ToggleButtons come in two sizes: **small** and **medium**. This allows for place
   </Hds::ButtonSet>
 </Doc::Layout>
 
+ToggleIcons come in two sizes: **small** and **medium**. 
+
+While we provide a small size variant, we recommend only using this for the Overflow menu within [Tables](/components/table) because the icons and images start to become unrecognizable in smaller sizes.
+
+<!-- TODO: Add small example -->
```

Comment:
@alex-ju Was the addition of the small ToggleIcon included in the code updates?

Reply:
Not available yet – I've raised https://github.com/hashicorp/design-system/pull/1276 to add this option – will need to rebase this PR after the other one gets merged.